doc/schema/attribute-types/enum.md

```diff
@@ -0,0 +1,76 @@
+# enumerate({default, primary, values, required, unique})
+
+Defines an attribute as being an enum type.
+
+```ts
+export const Todo: PartialSchema = {
+  name: "Todo",
+  attributes: {
+    status: enumerate({ values: ["Pending", "Failed", "Completed"] }),
+  },
+}
+```
+
+## Parameters
+
+| key          | description                                                                                      |   type    | optional |  default  |
+| ------------ | ------------------------------------------------------------------------------------------------ | :-------: | :------: | :-------: |
+| `default`    | The default value of the attribute. <br/> Example: `enumerate({default: "Pending", values:[...])`| `String`  |   Yes    |`undefined`|
+| `primary`    | If the attribute is a primary key. <br/> Example: `enumerate({primary: true, values:[...])`      | `Boolean` |   Yes    |  `false`  |
+| `values`     | The options of the enum <br/> Example: `enumerate({values: ["Pending", "Failed", "Completed"]})` | `Number`  |   No     |           |
```

Comment:
`type` here should be `Array` of type string? (or any?) idk what the backend supports exactly

Reply:
Thanks for catching that, it's definitely not number 😅